ansible/roles/go-cli/tasks/deploy.yml

```diff
@@ -6,15 +6,39 @@
     path: "{{ go_cli_conf_dir }}"
     state: directory
 
+- name: "Ensure nginx directory for go-cli exists"
+  file:
+    path: "{{ go_cli_nginx_dir  }}"
+    state: directory
+
 - name: "Pull the {{ docker_image_tag }} image of go-cli"
   shell: "docker pull {{ docker_registry }}whisk/go-cli:{{ docker_image_tag }}"
   when: docker_registry != ""
 
-# TODO: Publish compressed binaries using nginx
```

Comment:
We might have to arrange the compressed files better in the Docker container so the paths are not so long.